vendor/vendor.json

```diff
@@ -507,6 +507,12 @@
 			"revision": "b5461758462b97cff258bf388ca42110f529d747",
 			"revisionTime": "2017-11-07T15:27:38Z"
 		},
+		{
+			"checksumSHA1": "jHOLsfeMPcX84YQB2MXBqXFntBg=",
+			"path": "github.com/kr/pty",
```

Comment:
This package is licensed under MIT. 
Just comment to mark this.

In the future, we will list all the licenses of vendored package.

Reply:
Oh. I don't realize this. Sorry about that. 

It seems that we can do this without `pr/tty`. I will try.

Reply:
It is all right. Maybe in the future. We need these test very much. I think we could merge this ASAP.